integration-tests/src/test/resources/imagetool/create_image_base_no_internet.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+# Copyright 2019, Oracle Corporation and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+#  This script leverages the WebLogic image tool to build a WebLogic docker image with patches
+#  and without internet access. The version is 12.2.1.3.0 with PS4 and interim patch 2915930 and 30386660
+#  
+#  Assumptions:
+#    build_download.sh should be run first
+#    The $WIT_HOME_DIR working directory must have at least 10g of space.
+#
+#  Expects the following installers to already be installed:
+#    JDK Instaler: jdk-8u202-linux-x64.tar.gz to /scratch/artifacts/imagetool
+#    WebLogic Installer: fmw_12.2.1.3.0_wls_Disk1_1of1.zip to /scratch/artifacts/imagetool
+#    Patch 30386660_12.2.1.3.0 to /scratch/artifacts/imagetool
+#    Patch 28186730_13.9.4.0.0 to /scratch/artifacts/imagetool
+#    Patch 29135930_12.2.1.3.191004 to /scratch/artifacts/imagetool
+#
+#  Environment varibale http_proxy and https_proxy are not required
+#
+
+# Exit immediately if a command exits with a non-zero status.
+set -e
+
+cleanup()
+{
+  echo @@
+  echo "@@ Cleanup WIT cache Entry and old WLS docker image"
+  echo @@
+
+  # Clean WIT cache
```

Comment:
Would be safer to just "rm -rf" the cache folder.  The Image Tool will build a new one the next time it runs.